### ISSUE TITLE
isAppInFocus missing

### ISSUE THREAD

**Issue by alejandromagnorsky** [OneSignal/OneSignal-Cordova-SDK#102]
In the callback of handleNotificationOpened isAppInFocus is not present. 
Testing in iPhone 5, iOS 8.4. Didn't confirm with other versions.

Thanks!
Awesome tool ;)

**Comment by jkasten2** [comment]
@alejandromagnorsky Can you confirm you are using the latest 2.0.6 SDK by running the following.
```
cordova plugin list
```

Thanks.

**Comment by alejandromagnorsky** [comment]
@jkasten2 
Yes, here is the output:
`onesignal-cordova-plugin 2.0.6 "OneSignal Push Notifications"`

**Comment by aurbina83** [comment]
I'm having this same issue as of today.

  "version": "2.0.6",
  "name": "onesignal-cordova-plugin",
  "cordova_name": "OneSignal Push Notifications"

**Comment by jkasten2** [comment]
@alejandromagnorsky @aurbina83
[`OSNotification`](https://documentation.onesignal.com/docs/cordova-sdk#section--osnotification-) also contains `shown` which should be used know if you need to display the message in your app. Is this the use case you are looking for? We will still be looking into the `isAppInFocus` issue regardless.

**Comment by aurbina83** [comment]
I'm using isAppInFocus for a notificationReceivedHandler. 

            let notificationReceivedCallback = (data) => {
                if (data.payload.additionalData) {
                    let d = data.payload.additionalData;
                    if (d.type && d.type == "qrf" && data.isAppInFocus) {
                        this.alertQRF(data);
                    }
                } else return;
            }

I have to use isAppInFocus because if the user opens a notification, through the notificationOpenedCallback, the behavior is different.

                    if (d.type && d.type == "qrf") {
                        this.QrfService.getOne(d.id).then((data) => {
                            this.nav.push(QrfAcceptPage, { qrf: data });
                        }, (err) => {
                            console.log(err.message);
                        })
                    }

One is alerting (notificationReceived) and one is redirecting (notificationOpened), but without "isAppInFocus" if the user opens a notification, both get triggered. 

So far I haven't had any issues with Android, but as of today iOS isn't receiving data on isAppInFocus. 

In all honesty, until kinks are worked out, I'd like to just go back to the 1.x api but the documentation is gone. Is there a link to the old documentation somewhere?

**Comment by jkasten2** [comment]
@aurbina83 We don't have the old docs available but you can find the functions from our .js file in the last 1.X tag.
https://github.com/OneSignal/OneSignal-Cordova-SDK/blob/1.13.1/www/OneSignal.js

Thanks.

**Comment by aurbina83** [comment]
So it looks like the problem is on the handleNotificationReceived, the only thing that is getting passed in is the payload itself (on iOS).

Here is what I have.
```
let iosSettings = {};
iosSettings['kOSSettingsKeyAutoPrompt'] = true;
iosSettings['kOSSettingsKeyInAppLaunchUrl'] = false;

window['plugins'].OneSignal
    .startInit("30dda287-6c9b-42b6-a979-07089bfe9784", "531852131773")
    .inFocusDisplaying(window['plugins'].OneSignal.OSInFocusDisplayOption.None)
    .handleNotificationOpened(notificationOpenedCallback)
    .handleNotificationReceived(notificationReceivedCallback)
    .iOSSettings(iosSettings)
    .endInit();

let notificationReceivedCallback = (data) => {
    console.log(JSON.stringify(data));
    if (data.payload.additionalData) {
        let d = data.payload.additionalData;
        if (d.type && d.type == "qrf" && data.isAppInFocus) {
            this.alertQRF(data);
        }
    } else return;
}
```

And these are the responses I'm getting from both android and ios

Android: 

```
{
    "isAppInFocus": true,
    "shown": false,
    "androidNotificationId": 0,
    "displayType": 0,
    "payload": {
        "notificationID": "xxxxxxxxxxxxx",
        "title": "test",
        "body": "test",
        "additionalData": {
            "id": "xxxxxxxxxxxxx",
            "page": "QRFAcceptPage",
            "type": "qrf"
        },
        "lockScreenVisibility": 1,
        "fromProjectNumber": "xxxxxxxxxxxxx",
        "rawPayload": {
            "google.sent_time": xxxxxxxxxxxxx,
            "custom": {
                "a": {
                    "id": "xxxxxxxxxxxxx",
                    "page": "QRFAcceptPage",
                    "type": "qrf"
                },
                "i": "xxxxxxxxxxxxx"
            },
            "from": "xxxxxxxxxxxxx",
            "alert": "test",
            "title": "test",
            "google.message_id": "0:xxxxxxxxxxxxxxxxxxx",
            "collapse_key": "do_not_collapse"
        }
    }
}
```

iOS: 

```
{
    "payload": {
        "body": "test",
        "sound": "default",
        "title": "test",
        "notificationID": "xxxxxxxxxxxxx",
        "rawPayload": {
            "aps": {
                "alert": {
                    "title": "test",
                    "body": "test"
                },
                "sound": "default"
            },
            "custom": {
                "a": {
                    "id": "test",
                    "type": "qrf",
                    "page": "QRFAcceptPage"
                },
                "i": "xxxxxxxxxxxxx"
            }
        },
        "additionalData": {
            "id": "xxxxxxxxxxxxx",
            "type": "qrf",
            "page": "QRFAcceptPage"
        }
    }
}
```

So for iOS, no matter if it's "shown" or "isAppInFocus", it's not getting passed through.

**Comment by jkasten2** [comment]
@aurbina83 @alejandromagnorsky Just released 2.0.7 which includes a the `isAppInFocus` field for iOS.
